client/src/main/scala/com.gu.contentapi.client/model/package.scala

```diff
@@ -1,7 +1,7 @@
 package com.gu.contentapi.client
 
 package object model {
-  type PaginatedApiQuery[A <: Parameters[A]] = ContentApiQuery with PaginationParameters[A] with OrderByParameter[A]
```

Comment:
This is a key part of the PR - dropping the requirement for a content-api query to support `order-by` before we recognise it as a `PaginatedApiQuery` that we can use the pagination methods on!